README.md

```diff
@@ -370,6 +402,39 @@ let () =
   painIndexMap |> Hashtbl.iter (fun k v -> Js.log {j|key:$k, val:$v|j})
 ```
 
+###### Belt.HashMap
+
+Use a generic `Belt.HashMap` to demo a mutable hash map with string type key. In fact, You can use `Belt.HashMap.String` instead.
+
+```ml
+open Belt
+module StrHash =
+  Id.MakeHashable(struct
+                    type t = string
+                    let hash (x : t) = String.length x
+                    let eq (a : t) (b : t) = a = b
```

Comment:
Are all these type annotations really needed? It should be constrained by the signature MakeHashable expect, shouldn't it? Also, with partial application you should be able to just do `let hash = String.length` and `let eq = (=)`.